src/styles/components/statusbar.less

```diff
@@ -50,3 +50,11 @@
   }
 
 }
+
+.host-table {
+  .status-bar {
+    margin-right: 20px;
```

Comment:
This makes the bars smaller as the number gets larger. Do we want that?
See the right alignment of the bars being different here:
![](https://cl.ly/172H2N2J3V20/Image%202016-08-22%20at%2011.30.46.png)